csharp/src/Ice/IObjectPrx.cs

```diff
@@ -9,6 +9,13 @@
 
 namespace ZeroC.Ice
 {
+    /// <summary>A delegate that reads the response return value from a response frame.</summary>
+    /// <typeparam name="T">The type of the return value to read.</typeparam>
+    /// <param name="proxy">The proxy used to received the response frame.</param>
```

Comment:
A proxy does not receive a response frame. I think you meant the proxy used to send the request.